lib/travis/github_apps.rb

```diff
@@ -131,6 +134,10 @@ def fetch_new_access_token
       github_access_token
     end
 
+    def repositories_list
+      @repositories_list ||= repository_id.kind_of?(Array) ? repository_id.compact.reject(&:empty?).map(&:to_s) : [repository_id.to_s]
```

Comment:
Is it supposed to be an array? Or a single String, or something else? If it's an array, it shouldn't be named `repository_id`, which implies that it is a scalar value.

Reply:
It was possible to pass number or array, but you are right. We should keep it simple, so I refactored it.